Mapping.c

```diff
@@ -11,12 +11,11 @@ static unsigned short *make_id2insn(const insn_map *insns, unsigned int size)
 {
 	// NOTE: assume that the max id is always put at the end of insns array
 	unsigned short max_id = insns[size - 1].id;
-	unsigned short i;
 
 	unsigned short *cache =
 		(unsigned short *)cs_mem_calloc(max_id + 1, sizeof(*cache));
 
-	for (i = 1; i < size; i++)
+	for (unsigned int i = 1; i < size; i++)
```

Comment:
C99 requires to declare all variables at the beginning of the functions, or blocks.

we have been always doing this so far, so please avoid declaring variable inside for loop like this.